src/components/dashboard/assignments.tsx

```diff
@@ -0,0 +1,93 @@
+// Modules
+import Link from "next/link";
+
+// Components
+import Title from "@components/global/title";
+import Icon from "@components/global/icon";
+import Status from "@components/assignments/status";
+
+const AssignmentItem = ({
+  id,
+  name,
+  subject,
+  due,
+  status,
+  urgent,
+}: {
+  id: number;
+  name: string;
+  subject: string;
+  due: Date;
+  status: "not-started" | "started" | "done";
+  urgent: boolean;
+}): JSX.Element => {
+  return (
+    <li>
+      <Link href={`/assignments?id=${id}`}>
+        <a
+          className="flex flex-row justify-between items-end p-4 hover:bg-light-primary-0.08-tlc
+          focus:bg-light-primary-0.12-tlc active:bg-light-primary-0.12-tlc"
+        >
+          <div>
+            <p className="font-bold text-light-on-surface dark:text-dark-on-surface text-lg">{name}</p>
+            <p className="text-light-on-surface-variant dark:text-dark-on-surface-variant text-base">
+              {subject} • {due.toLocaleDateString("th-TH")}
```

Comment:
maybe use `moment` to format the time now in case we need to scale this up
or let timezone be an optional prop user might want to use